README.adoc

```diff
@@ -114,19 +114,19 @@ Spring Boot can configure large parts of typical applications based on the conte
 A single `@EnableAutoConfiguration` annotation triggers auto-configuration of the Spring context.
 
 Auto-configuration attempts to deduce which beans a user might need. For example, if `HSQLDB` is on the classpath, and the user has not configured any database connections, then they probably want an in-memory database to be defined.
-Auto-configuration will always back away as the user starts to define their own beans.
+Auto-configuration will always back away as the user starts to define their beans.
 
 
 
 === spring-boot-starters
 Starters are a set of convenient dependency descriptors that you can include in your application.
-You get a one-stop-shop for all the Spring and related technology you need without having to hunt through sample code and copy paste loads of dependency descriptors.
+You get a one-stopn shop for all the Spring and related technology you need without having to hunt through sample code and copy-paste loads of dependency descriptors.
```

Comment:
Looks like that extra `n` should be removed.

Reply:
Yes, the extra 'n' has been removed